packages/miniflare/src/plugins/pipelines/index.ts

```diff
@@ -0,0 +1,104 @@
+import { SharedBindings } from "miniflare:shared";
+import SCRIPT_PIPELINE_OBJECT from "worker:pipelines/pipeline";
+import { z } from "zod";
+import {
+	kVoid,
+	Service,
+	Worker_Binding_DurableObjectNamespaceDesignator,
+} from "../../runtime";
+import {
+	getMiniflareObjectBindings,
+	namespaceKeys,
+	objectEntryWorker,
+	Plugin,
+	ProxyNodeBinding,
+	SERVICE_LOOPBACK,
+} from "../shared";
+
+export const PipelineOptionsSchema = z.object({
+	pipelines: z.union([z.record(z.string()), z.string().array()]).optional(),
+});
+
+export const PIPELINES_PLUGIN_NAME = "pipelines";
+const SERVICE_PIPELINE_PREFIX = `${PIPELINES_PLUGIN_NAME}:pipeline`;
+const PIPELINE_OBJECT_CLASS_NAME = "Pipeline";
+const PIPELINE_OBJECT: Worker_Binding_DurableObjectNamespaceDesignator = {
+	serviceName: SERVICE_PIPELINE_PREFIX,
+	className: PIPELINE_OBJECT_CLASS_NAME,
+};
+
+export const PIPELINE_PLUGIN: Plugin<typeof PipelineOptionsSchema> = {
+	options: PipelineOptionsSchema,
+	getBindings(options) {
+		const pipelines = bindingEntries(options.pipelines);
+		return pipelines.map<Service>(([name, id]) => ({
+			name,
+			service: { name: `SERVICE_PIPELINE_PREFIX:${id}` },
```

Comment:
```suggestion
			service: { name: `${SERVICE_PIPELINE_PREFIX}:${id}` },
```

just a typo hopefully!

Reply:
🤦‍♂️